README.md

```diff
@@ -39,7 +40,7 @@ Lemmas: manual native
 UPOS: manual native
 XPOS: not available
 Features: manual native
-Relations: manual native
+Relations: converted from manual (initially annotated in the SUD format)
```

Comment:
This is machine-readable metadata line which must match one of the pre-defined strings. The part "(initially annotated in the SUD format)" cannot be here. But it can be described in the prose above, of course.